README.md

```diff
@@ -1 +1,20 @@
 # sis-qa-test-auto
+
+QA Test Automation framework for SIS Campus Solutions.
+
+
+### Instructions for running
+1. [Install Docker](https://docs.docker.com/installation/).
+2. If using external test directory, set `$CC_TEST_DIR` to location of external test directory.
+4. Add file test/.config.json with the following information:
+```json
+{
+    "website_url": "<Campus Solutions Test Instance URL>",
+    "userid": "<User ID for Test Instance>",
+    "password": "<Password for Test Instance>",
+    "title": "Berkeley Student Information System"
+}
+```
+3. Run `run.sh` (may required sudo. If using external test directory, run as `sudo -E ./run.sh` to 
```

Comment:
@mDibyo the numbering seems to be a bit messed up, could you check whether it works if we just use `1.` everywhere (even for the 2nd / 3th / ... items)?
Also, you might have to use 4 spaces for indenting the JSON - https://github.com/ets-berkeley-edu/calcentral/edit/master/README.md has some really good examples for this.